src/core.js

```diff
@@ -1242,6 +1246,10 @@ define([
             _this._resetMessages(e.errors);
         }).on('question-create', function (e) {
             _this.handleNewMug(e.mug, e.refMug, e.position);
+            var currentMug = _this.getCurrentlySelectedMug();
+            if (e.mug && e.mug.parentMug && e.mug.parentMug === currentMug) {
+                _this.displayMugProperties(currentMug);
+            }
             if (!e.isInternal) {
                 _this.jstree("deselect_all", true)
                      .jstree('select_node', e.mug.ufid);
```

Comment:
This one's necessary because when the two child options are auto-added to a new select, that should cause the type dropdown to change, but those events are internal, so the select_node doesn't get triggered.